pkg/urlutil/urlerrorhelper_test.go

```diff
@@ -0,0 +1,43 @@
+package urlutil
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+)
+
+func Test_urlparse01(t *testing.T) {
```

Comment:
please name the tests something other than Test_urlparse01, etc. For instance, this one could be Test_UrlRedacted or something else that lets us know what's being verified

Reply:
done